src/util/formatter.js

```diff
@@ -0,0 +1,48 @@
+
+const BEAUTIFY_SETTINGS = {
+  indent_size: 4,
+  indent_char: ' ',
+  indent_with_tabs: false,
+  eol: '\n',
+  end_with_newline: false,
+  indent_inner_html: true,
+  indent_level: 0,
+  preserve_newlines: true,
+  max_preserve_newlines: 10,
+  space_in_paren: false,
+  space_in_empty_paren: false,
+  jslint_happy: false,
+  space_after_anon_function: false,
+  brace_style: 'collapse',
+  unindent_chained_methods: false,
+  break_chained_methods: false,
+  keep_array_indentation: false,
+  unescape_strings: false,
+  wrap_line_length: 0,
+  e4x: false,
+  comma_first: false,
+  operator_position: 'before-newline',
+  /* don't add new lines before head,body,/html https://github.com/beautify-web/js-beautify */
+  extra_liners: [],
+};
+
+export function format(Beautify, code, mode, opts = {}) {
```

Comment:
I know this is kind of picky, but I’m wary of having two variable/parameter names which are basically “the same thing but one is abbreviated”. I would claim there is always a descriptive way to distinguish them! In this case I might call the parameter `optionsIn` or `optionOverrides`.